corehq/apps/translations/integrations/transifex/views.py

```diff
@@ -326,6 +328,13 @@ def perform_pull_request(self, request, form_data):
                                     'You should receive an email shortly'))
         return True
 
+    def perform_backup_request(self, request, form_data):
+        if not self.ensure_resources_present(request):
+            return False
+        backup_project_from_transifex.delay(request.domain, form_data, request.user.email)
+        messages.success(request, _('Successfully enqueued request to take backup.'))
+        return True
```

Comment:
You could move these form actions into the corresponding forms to keep things grouped together.  Feel free to disregard if that seems more complicated

Reply:
these methods do share code which would need to be moved too. Plus all of these work around the `request`, don't think being in the form is the right place for them. I would be fine with them being moved as utils or broken into classes under utils. Does that sound right to you?